executors/cmdexec/volume.go

```diff
@@ -69,6 +69,10 @@ func (s *CmdExecutor) VolumeCreate(host string,
 
 	commands = append(commands, s.createVolumeOptionsCommand(volume)...)
 
+	//Set many bricks per node volume parameters
+    commands = append(commands, fmt.Sprintf("gluster --mode=script volume %v set cluster.brick-multiplex on", volume.Name))
+    commands = append(commands, fmt.Sprintf("gluster --mode=script volume %v set transport.listen-backlog 100", volume.Name))
```

Comment:
Need to make this parameter configurable (environment)

Reply:
done